summer_staj.py

```diff
@@ -61,11 +71,14 @@ def start_requests(self):
             html = response.read()
             page = bs(html)
             # page.prettify()
-            rangeOfPages = page.find("span", {"style": "font-size:1.2em;"}).getText()
-
-
-        for site in range(int(rangeOfPages[4:])):
-            url = "http://mfstaj.cs.bilkent.edu.tr/visitor/?page=company&start=" + str(site) + "&filter=AllCompanies"

```

Comment:
.format is better instead of using +str(etc)+
In fact; check out python 3.6's f-strings